unipipe/blueprints/catalog.yml.js

```diff
@@ -129,39 +129,48 @@ export const catalog =
     plan_updateable: false
     instances_retrievable: true
     bindings_retrievable: false
+    metadata:
+      displayName: "Azure VNet Service"
+      imageUrl: "https://symbols.getvecta.com/stencil_28/71_virtual-network.4fe1e0cdfa.svg"
     plans:
       - id: ${uuid.generate()} # this uuid was randomly generated using "unipipe generate uuid"
-        name: default
-        description: Standard vNet
+        name: Standard
+        description: Standard vNet generates 1 Private and 1 Public Subnet
         free: true
         schemas:
           service_instance:
             create:
               parameters:
                 "$schema": http://json-schema.org/draft-04/schema#
                 properties:
-                  count_of_leading_1_bits_in_the_routing_mask:
-                    title: size of vNet
+                  VNetIP:
+                    title: vNet IP
+                    type: string
+                    widget:
+                      id: string
+                    default: "10.X.X.X"
+                  CidrBlock:
+                    title: CIDR Block
                     type: string
                     widget:
                       id: select
                     default: "28"
                     oneOf:
-                      - description: /27 vNet with 11 addresses (+5 addresses reserved by Azure)
+                      - description: /28 vNet with 6 addresses (+5 addresses reserved by Azure per each subnet)
+                        enum:
+                          - "28"
+                      - description: /27 vNet with 22 addresses (+5 addresses reserved by Azure per each subnet)
                         enum:
                           - "27"
-                      - description: /26 vNet with 27 addresses (+5 addresses reserved by Azure)
+                      - description: /26 vNet with 56 addresses (+5 addresses reserved by Azure per each subnet)
```

Comment:
d: The numbers don't add up. The /26 vnet has 64 addresses. Azure reserves 5 per subnet; so 10  addresses are reserved by the two subnets. This would leave us with 54 usable addresses within the two subnets (27 per subnet).

Reply:
this first and the last ip also unusable. should i also clarify this? it is more about networking not blocked by azure

Reply:
that is nice catch. thank you ! i also checked the other ones but it seems only this one was wrong. 
eg.  
- /28  = 16 ip - 10 = 6
- /27 = 32 ip - 10 = 22
- /24 = 256 ip - 10 = 246 

maybe im wrong but i will check while actually creating some of these resources